src/walltime.cc

```diff
@@ -92,7 +99,22 @@ class WallTimeImp
 
   WallTime Slow() const {
     struct timeval tv;
+#if defined(OS_WINDOWS)
+    FILETIME    file_time;
+    SYSTEMTIME  system_time;
+    ULARGE_INTEGER ularge;
+    const unsigned __int64 epoch = 116444736000000000LL;
+
+    GetSystemTime(&system_time);
+    SystemTimeToFileTime(&system_time, &file_time);
+    ularge.LowPart = file_time.dwLowDateTime;
+    ularge.HighPart = file_time.dwHighDateTime;
+
+    tv.tv_sec = (long)((ularge.QuadPart - epoch) / 10000000L);
```

Comment:
10000000L (and the entire expression) is integer, 1e7 is a double. What about 10L * 1000 * 1000 ?